comb_spec_searcher/specification_drawer.py

```diff
@@ -139,10 +143,25 @@ def _create_standard_tooltip(
             rule = self.spec.get_rule(comb_classes[-1])
         else:
             rule_string = str(comb_classes[0]).replace("\n", "<br>")
+
+        # labels
         labels = [str(self.spec.get_label(comb_class)) for comb_class in comb_classes]
         labels_string = ", ".join(labels)
+
+        # comb_class json
+        json_string = ""
+        if self.verbose:
+            comb_classes_json = [
+                "<br>JSON:<br>" + json.dumps(comb_class.to_jsonable())
+                for comb_class in comb_classes
+            ]
+            json_string = ", ".join(comb_classes_json)
+
         tooltip = {
-            "content": f"<p>Labels: {labels_string}</p><pre>{rule_string}</pre>",
+            "content": f"""
+                <p>Labels: {labels_string}</p>
+                <pre>{rule_string}</pre>
+                <p>{json_string}</p>""",
```

Comment:
Should the <p> and </p> be in json string so that in non verbose mode we don't add an empty paragraph?